www/src/pages/foundations/layout.mdx

```diff
@@ -10,20 +10,65 @@ import LayoutGenerator from '../../components/LayoutGenerator'
 
 # Layout
 
-
-
-## Grid 
+Paragon's layout is controlled by the Bootstrap grid system. Documentation can be found here:
+[https://getbootstrap.com/docs/4.5/layout/grid/](https://getbootstrap.com/docs/4.5/layout/grid/)
+
+
+## Components
+
+<p>
+  These components are pass throughs from React-Bootstrap.<br/>
+  <a href="https://react-bootstrap.github.io/components/layout/" target="_blank">
```

Comment:
I should have captured this on the main PR, but links with `_blank` should also have `rel="noopener noreferrer"`. might be worthwhile the `rel` attribute to those links in this PR, as well.

Reply:
oh right (why is the web broken)